RadarSDK/Include/Radar.h

```diff
@@ -185,6 +185,15 @@ typedef void (^_Nullable RadarBeaconCompletionHandler)(RadarStatus status, NSArr
  */
 typedef void (^_Nullable RadarTrackCompletionHandler)(RadarStatus status, CLLocation *_Nullable location, NSArray<RadarEvent *> *_Nullable events, RadarUser *_Nullable user);
 
+/**
+ Called when an encrypted track request succeeds, fails, or times out.
+
+ Receives the request status and, if successful, a JSON Web Token (JWT) containing an array of the events generated and the user. Decrypt the JWT server-side using your private key.
+
+ @see https://radar.com/documentation/sdk/ios
```

Comment:
should we be pointing to the iOS docs or fraud docs? comments above the callback and method are inconsistent at the moment

Reply:
Good catch, will update